### DISCUSSION TITLE
Can we have rdetr pretained mode?

### DISCUSSION THREAD

**Discussion by niiish32x** [ultralytics/ultralytics#2540] [General]
I can't find rtdetr model like  rtdetr_l.pt. There is only rtdetr model with suffix .pdparams.

**Comment by Laughing-q** [comment]
@niiish32x just use the following script and the weight will automatically download.
```python
from ultralytics.vit import RTDETR
model = RTDETR("rtdetr-x.pt")
```

**Reply by niiish32x** [reply]
Thanks a lot!  
By the way I hope  i can use ultralytics framework to train my own rtdetr mode one day.

**Reply by glenn-jocher** [reply]
@niiish32x you're welcome! We're glad to hear about your interest in training your own RT-DETR model. While we currently don't offer custom training for RT-DETR directly within the Ultralytics framework, we're continuously working on improving and expanding our capabilities. Keep an eye on our updates, and thank you for being part of the YOLO community! 🚀
